### ISSUE TITLE
When setting 'path' parameter 'dockerfile' parameter is also needed

### ISSUE THREAD

**Issue by tarikdem** [CircleCI-Public/aws-ecr-orb#64]
### Orb version

6.0.6
### What happened

I tried building and pushed an image to ECR which is in a subfolder in my repo by providing the `path` parameter (`path: './dir'`). That failed with:

```
#!/bin/bash -eo pipefail
docker_tag_args=""
IFS="," read -ra DOCKER_TAGS <<< "latest"
for tag in "${DOCKER_TAGS[@]}"; do
  docker_tag_args="$docker_tag_args -t $AWS_ACCOUNT_URL/repo:$tag"
done
docker build \
   \
  -f Dockerfile \
  $docker_tag_args \
  ./dir
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /home/circleci/project/Dockerfile: no such file or directory
Exited with code 1
CircleCI received exit code 1
```

It worked only when I added both `dockerfile: './dir/Dockerfile` and `path: './dir'`

### Expected behavior

Ideally, just by setting the path parameter the Docker context including Dockerfile should be set. If not, it is not stated in the docs that the dockerfile parameter is required when setting the path.

**Comment by sparta-developers** [comment]
We are also experiencing this. We are on version 6.6.0

```
docker_tag_args=""
IFS="," read -ra DOCKER_TAGS <<< "latest,${CIRCLE_SHA1}"
for tag in "${DOCKER_TAGS[@]}"; do
  docker_tag_args="$docker_tag_args -t $AWS_ECR_URL/${AWS_ECR_REPO}:$tag"
done
docker build \
  -f Dockerfile \
  $docker_tag_args \
  /home/circleci/project/half_dome
```

Looking at the [docker docs](https://docs.docker.com/engine/reference/commandline/build/#specify-a-dockerfile--f), it looks like the dockerfile parameter is distinct from the other path parameter.

```
$ cd /home/me/myapp/some/dir/really/deep
$ docker build -f /home/me/myapp/dockerfiles/debug /home/me/myapp
$ docker build -f ../../../../dockerfiles/debug /home/me/myapp
```

For what it's worth, we followed @tarikdem's suggestion and it worked for us.

**Comment by iynere** [comment]
thanks @tarikdem @sparta-developers ! looks like [this orb's `build-image` command](https://github.com/CircleCI-Public/aws-ecr-orb/blob/master/src/commands/build-image.yml) needs to be updated to match how we code this in our Docker orb:

https://github.com/CircleCI-Public/docker-orb/blob/master/src/commands/build.yml#L107-L119

feel free to submit a PR, should be fairly straightforward, otherwise we will get this taken care of shortly

**Comment by iynere** [comment]
patched in `circleci/aws-ecr@6.7.0`

**Comment by codebaard** [comment]
@tarikdem you saved my day. Had to deal with the exact same issue a whole day.
